### ISSUE TITLE
accuracy: scikit: Wrap sklearn.metrics accuracy methods within Accuracy plugins

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#733]
Add these to `model/scikit/dffml_model_scikit/accuracy/`

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

Depends: #732

**Comment by sk-ip** [comment]
this also looks interesting, https://scikit-learn.org/stable/modules/model_evaluation.html

**Comment by sk-ip** [comment]
Following documentation needs to be updated. when this is done.
1. https://intel.github.io/dffml/examples/flower17/flower17_scikit.html
